doc/source/examples/analysis/hydrogen_bonds/hbonds.ipynb

```diff
@@ -72,7 +72,15 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for frame, donor_ix, *_ in hbonds.results.hbonds:
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Fixed.</p>